<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslationWidget</class>
 <widget class="QWidget" name="TranslationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineFolder">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>591</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>/home/amenmd/myfs/source-codes/bilkon/go/src/github.com/yca/survey/questions/final/c</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>611</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="format">
    <string>%v of %m</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>591</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushPrev">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Prev</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushNext">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>240</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>77</y>
     <width>591</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushNextEmpty">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Empty</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
